README.md

```diff
@@ -149,6 +110,31 @@ The general steps to generate the data for predtimechart are:
 6. copy the contents of `../data` to your current folder
 7. add, commit, and push
 
+### Evals Data
+
+1. clone the dashboard repository and enter it (you only need the `predevals-config.yml`)
+2. clone the hub repository into `repo/`
+3. Run the container:
+  ```bash
+
+  repo="cdcepi/FluSight-forecast-hub/" # change this to where the oracle data should be fetched from
+  prefix="https://raw.githubusercontent.com/${repo}/refs/heads"
```

Comment:
Out of scope for this PR, but am curious: will `create-predevals-data.R` work for oracle data in S3?

Reply:
If the URL resolves to a text file, then yes.